File: M8/Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/wallet/WalletItem.java
package com.elgin.elginexperience.shipay.models.wallet;

import com.google.gson.annotations.SerializedName;

public class WalletItem {
    @SerializedName("item_title")
    public WalletItemItemTitle item_title;
    @SerializedName("quantity")
    public WalletItemQuantity quantity;
    @SerializedName("type")
    public WalletItemType type;
}

File: Exemplos/App_eXperience_Kotlin/Elgin_AppExperience_Kotlin-font/app/src/main/java/com/example/e1_kotlin_r/shipay/models/wallet/WalletItem.kt
package com.example.e1_elgin_kotlin.shipay.models.wallet

import com.google.gson.annotations.SerializedName

data class WalletItem (
    @SerializedName("item_title")
    var item_title: WalletItemItemTitle?,

    @SerializedName("quantity")
    var quantity: WalletItemQuantity,

    @SerializedName("type")
    var type: WalletItemType
)
